org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ActionImpl.java

```diff
@@ -220,8 +221,22 @@ public void stopRecording(String inputName) {
 		Changes useActiveChanges = activeChanges;
 		Logger useLogger = getLogger();
 		if (useLogger.isDebugEnabled()) {
-			useLogger.debug("Stop processing [ {} ] using [ {} ]: {}", inputName, getName(),
-				useActiveChanges.getChangeText());
+			useLogger.debug("Stop processing [ {} ] using [ {} ] took [ {}ms ]: {}", inputName, getName(),
+				useActiveChanges.getElapsedMillis(), useActiveChanges.getChangeText());
+
+		// let's try to provide a bit more information to the user for high level ActionTypes
+		} else if (useLogger.isInfoEnabled()
```

Comment:
This should be done somewhere in ZipActionImpl so all zip file types (EAR, RAR, WAR, JAR, ZIP) are covered. It is to brittle to use a list of action types here I think.

Reply:
Yes, I can do that